build-logic/convention/src/main/kotlin/MultiplaftormConventionPlugin.kt

```diff
@@ -0,0 +1,49 @@
+import com.android.build.gradle.LibraryExtension
+import org.gradle.api.JavaVersion
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
+import org.gradle.kotlin.dsl.configure
+
+class MultiplaftormConventionPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        with(target) {
+            with(pluginManager) {
+                apply("org.jetbrains.kotlin.multiplatform")
+                apply("com.android.library")
+
+                extensions.configure<KotlinMultiplatformExtension> {
+                    androidTarget {
+                        compilations.all {
+                            kotlinOptions {
+                                jvmTarget = "17"
+                            }
+                        }
+                    }
+
+                    listOf(
+                        iosX64(),
+                        iosArm64(),
+                        iosSimulatorArm64()
+                    ).forEach {
+                        it.binaries.framework {
+                            baseName = "shared"
+                            isStatic = true
+                        }
+                    }
+                }
+
+                extensions.configure<LibraryExtension> {
+                    compileSdk = 34
```

Comment:
can this be extracted to a central place?

Reply:
Done, added them to version catalog